docs/intro.rst

```diff
@@ -3,15 +3,25 @@
 Introduction
 ============
 
+*The relational model is still the most effective method*
+*of representing data. And SQL as a declarative,*
+*storage-neutral query language is powerful and versatile.*
+*We don’t need to abandon either.*
+*Instead, we build on what works in a language that affords* 
+*more power to its users, while being simpler and more consistent.*"
```

Comment:
I think to make it a proper quote you need to add whitespace to the beginning of every line, like this:

```suggestion
    The relational model is still the most effective method
    of representing data. And SQL as a declarative,
    storage-neutral query language is powerful and versatile.
    We don’t need to abandon either. Instead, we build on 
    what works in a language that affords more power to its 
    users, while being simpler and more consistent.
```